### PR TITLE
Testing1.0.1

### PR DESCRIPTION
**Pull request by Winterhart**
Work done by Daniel... He forgots to do a pull request 
Daniel Di Corpo
26331602

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,20 +67,6 @@ Issue the following command to run the app, the server will listen by default on
 
 You can use a proxy http server such as nginx or apache.
```
Comment (BenjaminTanguay) [inline README.md:69]: Why was this deleted from the README?

File: src/main/java/com/commafeed/backend/HttpGetter.java [hunk 2]
```diff
@@ -223,4 +223,11 @@ public void checkServerTrusted(X509Certificate[] arg0, String arg1) throws Certi
 			return null;
 		}
 	}
+
```
Comment (BenjaminTanguay) [inline src/main/java/com/commafeed/backend/HttpGetter.java:226]: Why is there a main method in a random file?

File: src/test/java/com/commafeed/backend/service/FeedUpdateServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		FeedUpdateService fus = mock(FeedUpdateService.class);
+		
+		Feed feed = mock(Feed.class);
+		FeedEntry feedEntry = mock(FeedEntry.class);
+		List<FeedSubscription> subscriptions = (List<FeedSubscription>) mock(FeedSubscription.class);
+		
+		feedEntry = new 
+		
+		//Expecting return false when feedEntry is null
+		when(fus.addEntry(feed, feedEntry, subscriptions)).thenReturn(false);
```
Comment (BenjaminTanguay) [inline src/test/java/com/commafeed/backend/service/FeedUpdateServiceTest.java:31]: This makes no sense. You mock the class that you are actually testing. You are saying: when this mock I created calls this method and those parameters, then return false.
Next line, you are saying assert that when this mock I created calls the method and the params mentioned above, it returns false. 
In short, you are testing mockito, not the addEntry method from FeedUpdateService.

**Review by BenjaminTanguay — CHANGES_REQUESTED** [review]
-Random files were changed (Gurk had the exact same problem for some reason)
- You are not testing the FeedUpdateService class. You mocked the class you were trying to test. No matter what changes you make to the FeedUpdateService class logic, the test will always pass.

**Comment by Winterhart** [comment]
We will cancel this request , as the code is not ready yet
